PocketKit/Sources/Sync/RecentSavesController.swift

```diff
@@ -0,0 +1,50 @@
+import Combine
+import CoreData
+
+
+public class RecentSavesController: NSObject {
+    @Published
+    private(set) public var recentSaves: [SavedItem]
+
+    public let itemChanged: PassthroughSubject<SavedItem, Never> = .init()
+    
+    private let space: Space
+    private let savedItemsController: NSFetchedResultsController<SavedItem>
+
+    init(space: Space) {
+        self.space = space
+        self.savedItemsController = space.makeItemsController()
+        try? savedItemsController.performFetch()
+        recentSaves = savedItemsController.fetchedObjects.flatMap({ Array($0.prefix(5)) }) ?? []
+        
+        super.init()
+        
+        savedItemsController.delegate = self
+    }
+}
+
+extension RecentSavesController: NSFetchedResultsControllerDelegate {
+    public func controllerDidChangeContent(_ controller: NSFetchedResultsController<NSFetchRequestResult>) {
+
+        if let _recentSaves = savedItemsController.fetchedObjects.flatMap({ Array($0.prefix(5)) }),
```

Comment:
```
        if let _recentSaves = savedItemsController.fetchedObjects.flatMap({ Array($0.prefix(5)) }),
```

^ this logic is reused, so it could be extracted into a computed property, for example, but a non-blocker.